docker/service/src/main/resources/set_resource_permissions_template.mustache

```diff
@@ -0,0 +1,98 @@
+      </device>
+    </communicationFunctionRcv>
+    <communicationFunctionSnd>
+      <device>
+        <id root="1.2.826.0.1285.0.2.0.107" extension="ZZZ000-100000000800001"/>
+      </device>
+    </communicationFunctionSnd>
+    <wsa:ReplyTo>
+      <wsa:Address>http://myMSH.com/pds/myPage.asmx</wsa:Address>
+    </wsa:ReplyTo>
+  </SOAP-ENV:Header>
+  <SOAP-ENV:Body>
+    <SET_RESOURCE_PERMISSIONS_INUK01 type="Message" xmlns="urn:hl7-org:v3" xmlns:crs="http://national.carerecords.nhs.uk/schema/crs/">
+      <id root="11111111-1111-1111-1111-111111111111"/>
+      <creationTime value="200401061130"/>
+      <versionCode code="V3NPfIT3.0"/>
+      <interactionId root="2.16.840.1.113883.2.1.3.2.4.12" extension="SET_RESOURCE_PERMISSIONS_INUK01"/>
+      <processingCode code="P"/>
+      <processingModeCode code="T"/>
+      <acceptAckCode code="NE"/>
+      <communicationFunctionRcv type="CommunicationFunction" typeCode="RCV">
+        <device classCode="DEV" determinerCode="INSTANCE" type="Device">
+          <id root="1.2.826.0.1285.0.2.0.107" extension="ZZZ999-100000000900001"/>
+        </device>
+      </communicationFunctionRcv>
+      <communicationFunctionSnd type="CommunicationFunction" typeCode="SND">
+        <device type="Device" classCode="DEV" determinerCode="INSTANCE">
+          <id root="1.2.826.0.1285.0.2.0.107" extension="ZZZ000-100000000800001"/>
+        </device>
+      </communicationFunctionSnd>
+      <ControlActEvent classCode="CACT" moodCode="EVN">
+        <author typeCode="AUT">
+          <AgentPersonSDS classCode="AGNT">
+            <id root="1.2.826.0.1285.0.2.0.67" extension="7643324387"/>
+            <agentPersonSDS classCode="PSN" determinerCode="INSTANCE">
+              <id root="1.2.826.0.1285.0.2.0.65" extension="5452322"/>
+            </agentPersonSDS>
+            <part typeCode="PART">
+              <partSDSRole classCode="ROL">
+                <id root="1.2.826.0.1285.0.2.1.104" extension="ABCD"/>
+              </partSDSRole>
+            </part>
+          </AgentPersonSDS>
+        </author>
+        <author1 typeCode="AUT">
+          <AgentSystemSDS classCode="AGNT">
+            <agentSystemSDS classCode="DEV" determinerCode="INSTANCE">
+              <id root="1.2.826.0.1285.0.2.0.107" extension="accredited system id"/>
+            </agentSystemSDS>
+          </AgentSystemSDS>
+        </author1>
+        <author1 typeCode="AUT">
+          <AgentSystemSDS classCode="AGNT">
+            <agentSystemSDS classCode="DEV" determinerCode="INSTANCE">
+              <id root="9.99.999.9.999999.9.9.9.9.9.99" extension="workstation id"/>
+            </agentSystemSDS>
+          </AgentSystemSDS>
+        </author1>
+        <crs:setResourcePermissionsRequest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://national.carerecords.nhs.uk/schema/crs/ ..\schema\setResourcePermissionsRequest_0_4.xsd">
```

Comment:
I know this file is copied from the doc, but I'm pretty sure xsi:schemaLocation is syntactically incorrect.